main.css

```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+This is an example to execute the assignment 2 on Git repositories.
```

Comment:
Can you please add  more lines in this file

Reply:
lines have been added.